README.rst

```diff
@@ -48,20 +46,23 @@ Resources
 
 Here's a quick summary of resources to help you find your way around:
 
+* **Help**: `Asking for Help Tips`_
 * **Documentation**: http://docs.zephyrproject.org (`Getting Started Guide`_)
 * **Source Code**: https://github.com/zephyrproject-rtos/zephyr is the main
   repository; https://elixir.bootlin.com/zephyr/latest/source contains a
   searchable index
-* **Releases**: https://zephyrproject.org/developers/#downloads.
+* **Releases**: https://zephyrproject.org/developers/#downloads
 * **Samples and example code**: see `Sample and Demo Code Examples`_
 * **Mailing Lists**: users@lists.zephyrproject.org and
   devel@lists.zephyrproject.org are the main user and developer mailing lists,
   respectively. You can join the developer's list and search its archives at
   `Zephyr Development mailing list`_. The other `Zephyr mailing list
   subgroups`_ have their own archives and sign-up pages.
+* **Nightly CI Build Failures**: https://lists.zephyrproject.org/g/builds
```

Comment:
Not failures, this can also be all passes, use "Status" instead.

Reply:
doh.  I thought I had changed that.  New version pushed.